sql/moz-fx-data-shared-prod/telemetry_derived/desktop_eng_agg_v1/query.sql

```diff
@@ -0,0 +1,31 @@
+SELECT
+  submission_date,
+  first_seen_date,
+  attribution_campaign,
+  attribution_content,
+  attribution_dlsource,
+  attribution_medium,
+  attribution_ua,
+  normalized_os,
+  os_version_major,
+  country,
+  lifecycle_stage,
```

Comment:
the aggregate table should also have this computed at the view level, in case we need to modify it later.

Reply:
In the "desktop engagement aggregate" table, this code is just pulling the previously calculated "lifecycle stage" from the previous view (desktop engagement client), so whenever the other view lifecycle definition is  modified, that will flow through to this aggregate table automatically